browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3227,6 +3227,18 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_TRIAL" desc="Brave VPN trial text">
       Trial
     </message>
+    <message name="IDS_DISABLE_BRAVE_VPN" desc="Brave VPN timer action text">
+      Disable Brave VPN
+    </message>
+    <message name="IDS_PAUSE_FOR_15_MINUTES" desc="Brave VPN timer action text">
+      Pause for 15 minutes
+    </message>
+    <message name="IDS_PAUSE_FOR_1_HOUR" desc="Brave VPN timer action text">
+      Pause for 1 hour
+    </message>
+    <message name="IDS_DISABLE" desc="Brave VPN timer action text">
```

Comment:
Might be better to have a more descriptive `name`, e.g. `IDS_VPN_DISABLE`.

Reply:
@mkarolin i thought of keeping `Disable` as generic text. we may use it in future and we already have `DISABLE_BRAVE_VPN` as title.

Reply:
Sure, that's cool.